_posts/2016-12-30-foreman-community-newsletter---december-2016.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Foreman Community Newsletter - December 2016
+date: 2016-12-28 12:42:23
+author: Ondřej Pražák
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Foreman 1.14, Katello 3.3, plugin updates, YouTube content, new releases...
+  in the very last Foreman blog post of 2016.
+---
+
+### Foreman 1.14 branched
+
+Foreman 1.14 was branched on the 9th Dec. As always, a new version is
+packed with exciting features and important bugfixes. You can read about the details in the [announcement][1_14]
+
+[1_14]: https://groups.google.com/forum/#!topic/foreman-dev/RVtYtS5I7pU
+
+### Katello 3.2.1 released
+
+Katello 3.2.1 was released on 15th Dec. The most notable improvements are preformance enhancemnts for a capsule sync,
+fix for capsule install issues for Puppet 4 users and using newer Pulp version that takes care of removing sync issues
+for EPEL on EL7. More details can be found in the [announcement][k3_2_1]
+
+[k3_2_1]: https://groups.google.com/forum/#!topic/foreman-users/m_ZfV9RsIgU
+
+### Katello 3.3 release
+
+
+Katello 3.3 was released on the ??th Dec, [announcement][k3_3].
+
+[k3_3]:
+
+### New media and blogs
+
+TODO: None yet!
+
+#### Community Demos
+
+* [Community Demo - 15th December](https://www.youtube.com/watch?v=CY1U_54Cbbs)
+
+The latest instalment of the community demo series showcases all that's
+happening in the Foreman community. In the last demo we covered UI background tasks, ignoring facts on import,
+new Ansible options and more!
```

Comment:
Make a big point of the panel at the end, I think it was the best part of this recording, and may spark some discussion if we're lucky :)